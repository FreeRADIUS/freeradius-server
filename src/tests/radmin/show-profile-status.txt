#
#  PRE: set-profile-status-yes
#
show profile status
