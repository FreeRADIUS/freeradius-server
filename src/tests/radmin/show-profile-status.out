on /tmp/tapioca-com-queijo
